rest-api/dao/participant_summary_dao.py

```diff
@@ -369,7 +384,11 @@ def update_enrollment_status(self, summary):
     the correct value based on the other fields on it. Called after
     a questionnaire response or physical measurements are submitted."""
     consent = (summary.consentForStudyEnrollment == QuestionnaireStatus.SUBMITTED and
-               summary.consentForElectronicHealthRecords == QuestionnaireStatus.SUBMITTED)
+               summary.consentForElectronicHealthRecords == QuestionnaireStatus.SUBMITTED) or \
+              (summary.consentForStudyEnrollment == QuestionnaireStatus.SUBMITTED and
+               summary.consentForElectronicHealthRecords is None and
+               summary.consentForDvElectronicHealthRecordsSharing == QuestionnaireStatus.SUBMITTED)
+
```

Comment:
update the consent condition for dv ehr sharing

Reply:
we should change this so that either or both types of consent are acceptable.

Reply:
This is already covered all the conditions, isn't it?

Reply:
discussed offline, this is ok.

Reply:
Explain a little more about why we need to check `summary.consentForElectronicHealthRecords is None`. We have a discussion about this with before, here is the conclusion:
If they expressed intent as a DV, but ultimately declined in the EHR consent, then the decline would control status. So the logic is EHR consent is yes or (EHR consent is not set and DV EHR sharing is yes)